README.md

```diff
@@ -54,21 +45,31 @@ Snippets for deployment template and 70+ Azure resources.
 
 ## Insert Item
 
-In addition to adding snippets from the code editor, the Insert Item feature can be used to insert new parameters, user-defined functions, variables, resources, and outputs. To do so right-click on the template in the code editor, select **Insert Item**, and follow the on-screen prompts.
+In addition to adding snippets from the code editor, the Insert Item feature can be used to insert new parameters, user-defined functions, variables, resources, and outputs. To do so right-click on the template in the code editor, select **Insert Item** and follow the on-screen prompts.
 
 ![Inserting an item into an Azure Resource Manager template](./images/insert-item.png)
 
-You can also right-click on any element in the ARM Template Outline to initiate the insert item process.
+You can also right-click on any element in the ARM template Outline to initiate the insert item process.
```

Comment:
```suggestion
You can also right-click on any element in the ARM Template Outline view to initiate the insert item process.
```

Reply:
Or get rid of both capitals.  This is how it was mentioned above.

Reply:
Fixed